node/parallel-dev/src/chain_spec.rs

```diff
@@ -207,32 +208,57 @@ fn testnet_genesis(
                 .collect(),
         },
         loans: LoansConfig {
-            currencies: vec![CurrencyId::KSM, CurrencyId::USDT, CurrencyId::xKSM],
             borrow_index: Rate::one(),                             // 1
             exchange_rate: Rate::saturating_from_rational(2, 100), // 0.02
-            base_rate: Rate::saturating_from_rational(2, 100),     // 2%
-            jump_rate: Rate::saturating_from_rational(10, 100),    // 10%
-            full_rate: Rate::saturating_from_rational(32, 100),    // 32%
-            jump_utilization: Ratio::from_percent(80),             // 80%
-            collateral_factor: vec![
-                (CurrencyId::KSM, Ratio::from_percent(50)),
-                (CurrencyId::USDT, Ratio::from_percent(50)),
-                (CurrencyId::xKSM, Ratio::from_percent(50)),
-            ],
-            liquidation_incentive: vec![
-                (CurrencyId::KSM, Rate::saturating_from_rational(110, 100)),
-                (CurrencyId::USDT, Rate::saturating_from_rational(110, 100)),
-                (CurrencyId::xKSM, Rate::saturating_from_rational(110, 100)),
-            ],
-            close_factor: vec![
-                (CurrencyId::KSM, Ratio::from_percent(50)),
-                (CurrencyId::USDT, Ratio::from_percent(50)),
-                (CurrencyId::xKSM, Ratio::from_percent(50)),
-            ],
-            reserve_factor: vec![
-                (CurrencyId::KSM, Ratio::from_percent(15)),
-                (CurrencyId::USDT, Ratio::from_percent(15)),
-                (CurrencyId::xKSM, Ratio::from_percent(15)),
+            markets: vec![
+                (
+                    CurrencyId::KSM,
+                    Market {
+                        close_factor: Ratio::from_percent(50),
+                        collateral_factor: Ratio::from_percent(50),
+                        liquidate_incentive: Rate::from_inner(Rate::DIV / 100 * 110),
+                        state: MarketState::Active,
+                        rate_model: InterestRateModel::Jump(JumpModel::new_model(
+                            Rate::from_inner(Rate::DIV / 100 * 2),
+                            Rate::from_inner(Rate::DIV / 100 * 10),
```

Comment:
it seems like `Rate::from_inner(Rate::DIV / 100 * 2)`  works like `Rate::saturating_from_rational(2, 100)`, do they have any difference?

Reply:
`Rate::from_inner` is a context function but this feature does not matter in this context. Changed back to `saturating_from_rational`